vaadin-time-picker-flow/src/main/java/com/vaadin/flow/component/timepicker/GeneratedVaadinTimePicker.java

```diff
@@ -921,6 +929,6 @@ public GeneratedVaadinTimePicker(T initialValue, T defaultValue,
      */
     public GeneratedVaadinTimePicker() {
         this(null, null, null, (SerializableFunction) null,
-                (SerializableFunction) null);
+                (SerializableFunction) null, true);
```

Comment:
The rule here is to use `isInitialValueOptional=true` when constructor has no init value in arguments as in this particular case, right? `false` otherwise. Is so, just forget my comment.

Reply:
That's a mistake.
Thanks.

Reply:
Although it doesn't matter which value is used here.
`null`  as initial value makes it extra.